spec/forms/generic_file_edit_form_spec.rb

```diff
@@ -1,11 +1,9 @@
 require 'spec_helper'
 
 describe Hydranorth::Forms::GenericFileEditForm do
-  subject { described_class.new(GenericFile.new) }
-
   describe "#terms" do
```

Comment:
This is an extremely poor test. #terms is a method being inherited from the presenter class. The presenter test should be testing this logic (which we have no tests for), and GenericFiledEditForm specific tests should be in this file?